Dockerfile

```
@@ -0,0 +1,16 @@
+FROM golang:1.13 AS builder
+
+RUN mkdir /workdir
```

> > Sounds like you're having the same problem as @madmath03. We could just add Python explicitly just in case.
> 
> yes, I think too.....where should I add "python" ?

@rgl1234 & @gary-kim I think this should do the trick:

```suggestion
RUN set -ex; \
    mkdir /workdir; \
    apt-get update; \
    apt-get install -y --allow-unauthenticated --no-install-recommends \
        python3 \
    ;
```

---

thanks.....this worked for me...